containers/ovs-dbg/start.sh

```diff
@@ -17,7 +17,14 @@ vswitchd-dummy() {
   fi
   ) &
 
-  ovs-vswitchd --enable-dummy=override -vvconn -vnetdev_dummy  --no-chdir --pidfile  -vsyslog:off unix:${OVSDB_SOCKET}
+  if [[ "$UID" -eq 0 ]]; then
```

Comment:
Maybe also use root if "$UID" is not set?

Reply:
Right, change is made